opencog/learning/PatternMiner/DistributedPatternMinerClient.cc

```diff
@@ -23,8 +23,8 @@
  */
 
 #include "DistributedPatternMinerClient.h"
-#include <opencog/server/CogServer.h>
-#include <opencog/server/Factory.h>
+#include <opencog/cogserver/server/CogServer.h>
+#include <opencog/cogserver/server/Factory.h>
 #include <opencog/util/Logger.h>
```

Comment:
strange .. is this not built by default? I did not see this break, when I made the changes.